crates/board/src/crypto.rs

```diff
@@ -195,60 +202,49 @@ pub trait LastError {
 
 /// Wrapper API around Hash that calls last_error.
 #[cfg(feature = "internal-api-crypto-hash")]
-pub struct HashApi<T: Hash>(pub T);
+pub struct HashApi<T: Hash>(T);
 
 /// Wrapper API around Hmac that calls last_error.
 #[cfg(feature = "internal-api-crypto-hmac")]
-pub struct HmacApi<T: Hmac>(pub T);
-
-#[cfg(feature = "internal-api-crypto-hash")]
-impl<T: Hash> OutputSizeUser for HashApi<T> {
-    type OutputSize = T::OutputSize;
-}
-
-#[cfg(feature = "internal-api-crypto-hmac")]
-impl<T: Hmac> OutputSizeUser for HmacApi<T> {
-    type OutputSize = T::OutputSize;
-}
+pub struct HmacApi<T: Hmac>(T);
 
 #[cfg(feature = "internal-api-crypto-hash")]
 impl<T: Hash> HashApi<T> {
-    /// Constructor of HmacApi.
+    /// Create a hash wrapper.
```

Comment:
nit: this project follows [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Function_Comments) for function documentation
```suggestion
    /// Creates a hash wrapper.
```
(similarly for the other functions below)